content/community/team.md

```diff
@@ -8,6 +8,7 @@ This page lists who we are. By all means add yourself to the list - lets sort it
 
 When posting to the mailing lists, use plain text mails. Do not use HTML mails. HTML mails is more likely to be targeted as spam mails and will be rejected; as well it's not easily readable by others.
 
+{{< div "table-wrapper" >}}
```

Comment:
As far as I understand we don't need this anymore

Reply:
Yes, I changed it to directly include table wrapper within the table syntax in the shortcodes.